test/features/deploy_quarkus_service.feature

```diff
@@ -51,6 +51,31 @@ Feature: Deploy quarkus service
       | native  | minutes |
       | enabled | 20      |
 
+#####
+
+  Scenario Outline: Deploy process-optaplanner-quarkus service without persistence and native <native>
+    Given Kogito Operator is deployed
+    When Deploy quarkus example service "process-optaplanner-quarkus" with configuration:
+      | config | native | <native> |
+    Then Kogito application "process-optaplanner-quarkus" has 1 pods running within <minutes> minutes
+    And HTTP POST request on service "process-optaplanner-quarkus" is successful within 2 minutes with path "rest/flights" and body:
+      """json
+      {
+        "params" : {
+          "origin" : "A",
+          "destination" : "B",
+          "departureDateTime" : "2020-05-30T17:30:43.873968",
+          "seatRowSize" : 6,
+          "seatColumnSize" : 10
+        }
+      }
+      """
+
+    @native
```

Comment:
missing non-native examples here

Reply:
I was thinking of doing just a single combination in order not to prolong the execution time too much. Is it required to include the non-native variant? If so, I will add it, no problem.